src/wrappers/_ContentLoaded.jsx

```diff
@@ -7,25 +7,28 @@ import styles from './ContentLoaded.module.scss';
 const cx = classNames.bind(styles);
 
 const propTypes = {
+  className: PropTypes.string,
   children: PropTypes.element.isRequired,
 };
 
-const ContentLoaded = ({ children }) => {
+const ContentLoaded = ({ children, className }) => {
   useEffect(() => {
     if (!window.location || window.location.length < 2) {
       return;
     }
     const elementName = window.location.hash.slice(1);
-    const element = document.getElementsByName(elementName);
-    if (element[0]) {
-      element[0].scrollIntoView();
+    console.log('elementName', elementName);
+    const element = document.getElementById(elementName);
+    console.log('element', element);
+    if (element) {
+      element.scrollIntoView();
     }
-  });
+  }, []);
   return (
     <div
       id="site"
       data-terra-dev-site-content
-      className={cx('dev-site-content')}
+      className={`${cx('dev-site-content')} ${className}`}
```

Comment:
Should we be binding these like:
```
className={cx(['dev-site-content', className])}
```

Reply:
This is 100% on purpose. `className` is a custom class that is appended to this component by it's consumer. We should not search the class styles defined by this component for the class name provided by the consumer. Either the consumer can append some erroneous class name to the div that was not intended for this div or more likely we waste time trying to find the classname when it doesn't exist. This is wrong everywhere we do it and only works because cx returns the class name un modified if a match isn't found.